README.rst

```diff
@@ -122,6 +122,14 @@ Contributions are welcome for both code and localizations (adding and updating l
 1. Find an issue or feature to tackle on the `issue tracker <https://github.com/crsmithdev/arrow/issues>`_. Issues marked with the `"good first issue" label <https://github.com/crsmithdev/arrow/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22>`_ may be a great place to start!
 2. Fork `this repository <https://github.com/crsmithdev/arrow>`_ on GitHub and begin making changes in a branch.
 3. Add a few tests to ensure that the bug was fixed or the feature works as expected.
-4. Submit a pull request and await feedback 😃.
+4. Run the entire test suite and linting checks by running one of the following commands:
```

Comment:
It seems to render this line in bold.

Reply:
Seems to be a glitch with Github's PR preview:
<img width="774" alt="Screen Shot 2020-05-03 at 1 06 02 PM" src="https://user-images.githubusercontent.com/7388203/80920604-e2c0db00-8d3e-11ea-8d39-1212b660420d.png">

Reply:
Huh that's weird.